app/posts/cross-programme/2024-06-19-induction-discovery-recap.md

```diff
@@ -54,9 +42,9 @@ We then plotted a To-be journey that collects the data in a less burdensome way
 
 ## What did we learn? 
 
-As well as developing our understanding of the As is – in doing so we uncovered the flaws and bugs with the current processes of managing induction data: 
+As well as developing our understanding of the As is, we uncovered the flaws and bugs with the current processes of managing induction data: 
 
-* There are known issues in the back end where certain data combinations cause unwanted record changes. Combined with front-end validation rules that are not complete, this has resulted in inaccurate data being created in the DQT which is time consuming to correct. 
+* there are known issues in the back end where certain data combinations cause unwanted record changes. Combined with front-end validation rules that are not complete, this has resulted in inaccurate data being created in the DQT which is time consuming to correct. 
```

Comment:
@aabdi18 Hey! It's worth looking at the styling advice for bullet points in the GOVUK content advice here, under 'B': https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style

I don't think it matters too much for design histories, but good to be aware of for future design work that touches the service.